### PR TITLE
Theme Changes

### PR DESCRIPTION
**Pull request by manyaagarwal**
1. Added Theme Colours 
2. Added Status Bar

### CODE REVIEW

File: mobile/App.tsx [hunk 1]
```diff
@@ -1,21 +1 @@
...
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fff",
    alignItems: "center",
    justifyContent: "center",
  },
});
```
Comment (taneliang) [inline mobile/App.tsx:1]: Oof this got messed up 😆

File: mobile/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "@react-navigation/native": "^5.5.0",
     "expo": "~37.0.3",
     "expo-updates": "~0.2.0",
+    "format": "^0.2.2",
```
Comment (taneliang) [inline mobile/package.json:19]: This doesn't look right. If you're trying to run Prettier, use `yarn format`, which will run the `format` script defined above on line 9 (which runs Prettier)

File: mobile/App.tsx (hunk 1 above)
Comment (manyaagarwal) [reply mobile/App.tsx]: Changed newlines and removed format! 😅

File: mobile/yarn.lock [hunk 3]
```diff
@@ -3339,6 +3339,11 @@ form-data@~2.3.2:
     combined-stream "^1.0.6"
     mime-types "^2.1.12"
 
+format@^0.2.2:
```
Comment (taneliang) [inline mobile/yarn.lock:3342]: I think you need to run Yarn to remove this. I can do it

**Pull request merged**